### PR TITLE
[DOCS] Adds 7.3.0 release highlights

### PR DESCRIPTION
**Pull request by lcawl**
This PR adds a 7.3.0 release highlights section to the APM Overview (https://www.elastic.co/guide/en/apm/get-started/7.3/apm-release-notes.html)

### CODE REVIEW

**Comment by lcawl** [comment]
These changes can be previewed here: http://apm-server_2516.docs-preview.app.elstc.co/guide/en/apm/get-started/7.3/release-highlights-7.3.0.html

File: docs/guide/apm-release-notes.asciidoc [hunk 1]
```diff
@@ -23,6 +24,67 @@ For a full list of changes, see the
...
+These charts are available in Beta with support for certain APM agents(Java, Go,
+Node.js, Python).
+
+[role="screenshot"]
+image::images/apm-highlight-breakdown-charts.png[Aggregate service breakdown charts]
+
+[discrete]
+==== Agent sample rate configuration from APM UI
+
+Configuring sampling rate for your services is a whole lot easier with this
```
Comment (jalvz) [inline docs/guide/apm-release-notes.asciidoc]: @nehaduggal this a Beta feature as well, right? (asking because of missing `beta[]`)

Comment (lcawl) [inline docs/guide/apm-release-notes.asciidoc]:
```suggestion
beta[] Configuring sampling rate for your services is a whole lot easier with this
```

Comment (lcawl) [reply docs/guide/apm-release-notes.asciidoc]: Thanks, I've implemented that change.

File: docs/guide/apm-release-notes.asciidoc [hunk 2]
```diff
@@ -23,6 +24,67 @@ For a full list of changes, see the
...
+
+[role="screenshot"]
+image::images/apm-highlight-breakdown-charts.png[Aggregate service breakdown charts]
+
+[discrete]
+==== Agent sample rate configuration from APM UI
+
+beta[] Configuring sampling rate for your services is a whole lot easier with this
+release. The new settings page now lets you view and configure the sampling rate
+for your services from within the APM UI without changing the agent yaml files
```
Comment (jalvz) [inline docs/guide/apm-release-notes.asciidoc]: The agents don't have yaml config files. I suggest replace:

`for your services from within the APM UI without changing the agent yaml files
for each individual reporting instance of the service` ->
`for your services from within the APM UI without restarting them.` 
or something like that

Comment (lcawl) [reply docs/guide/apm-release-notes.asciidoc]: Done, thanks!

**Pull request merged**